### PR TITLE
Use Arcade NuSpec helpers

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.nuspec [hunk 1]
```diff
@@ -1,17 +1,7 @@
...
-    <authors>$authors$</authors>
-    <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <licenseUrl>$licenseUrl$</licenseUrl>
-    <projectUrl>$projectUrl$</projectUrl>
-    <iconUrl>$iconUrl$</iconUrl>
-    <description>$description$</description>
-    <copyright>$copyright$</copyright>
-    <tags>$tags$</tags>
-    <repository type="git" url="$repositoryUrl$" commit="$repositoryCommit$" />
+    $CommonMetadataElements$
```
Comment (dougbu) [inline src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.nuspec]: Just to confirm: The Arcade helpers i.e. the `InitializeStandardNuspecProperties` target does the `$CommonMetadataElements$` substitution?

Comment (tmat) [reply src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.nuspec]: Yup: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.targets#L102

**Comment by tmat** [comment]
@dougbu The CI is failing with some provisioning error:
image

The Windows Release build failure seems to be caused by missing `-configuration $(_BuildConfig)` argument to `eng\scripts\ci-flaky-tests.cmd`. The leg builds Release but attempts to run tests from Debug dir.

**Comment by dougbu** [comment]
> The Windows Release build failure seems to be caused by missing -configuration $(_BuildConfig) argument to eng\scripts\ci-flaky-tests.cmd. The leg builds Release but attempts to run tests from Debug dir.

That's odd. These test legs have been building and testing just fine up to now. Do the new Arcade scripts effectively make the build configuration a mandatory input?

In any case, we're still hitting dotnet/core-eng#5654 for the Ubuntu jobs&hellip;

----
How did you get the extra information beyond the horrible "There as a failure" message?

**Comment by tmat** [comment]
> That's odd. These test legs have been building and testing just fine up to now. Do the new Arcade scripts effectively make the build configuration a mandatory input?

IDK. It's always defaulted to Debug. I'm not aware of any change in that. Is it possible that he machines were reused and had the Debug build on them accidentally?

**Comment by dougbu** [comment]
Merged these changes manually into `darc`'s branch. Will take this further in #1279. Thanks @tmat❕❕

**Pull request closed without merging**